include/bluetooth/audio.h

```diff
@@ -36,6 +36,25 @@ struct bt_audio_cap;
 
 #define BT_BROADCAST_ID_SIZE             3 /* octets */
 
+/* Audio Context Type, Generic Audio */
+#define BT_AUDIO_CONTEXT_TYPE_PROHIBITED         BIT(0)
+#define BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED        BIT(1)
+#define BT_AUDIO_CONTEXT_TYPE_CONVERSATIONAL     BIT(2)
+#define BT_AUDIO_CONTEXT_TYPE_MEDIA              BIT(3)
+#define BT_AUDIO_CONTEXT_TYPE_GAME               BIT(4)
+#define BT_AUDIO_CONTEXT_TYPE_INSTRUCTIONAL      BIT(5)
+#define BT_AUDIO_CONTEXT_TYPE_VOICE_ASSISTANTS   BIT(6)
+#define BT_AUDIO_CONTEXT_TYPE_LIVE               BIT(7)
+#define BT_AUDIO_CONTEXT_TYPE_SOUND_EFFECTS      BIT(8)
+#define BT_AUDIO_CONTEXT_TYPE_NOTIFICATIONS      BIT(9)
+#define BT_AUDIO_CONTEXT_TYPE_RINGTONE           BIT(10)
+#define BT_AUDIO_CONTEXT_TYPE_ALERTS             BIT(11)
+#define BT_AUDIO_CONTEXT_TYPE_EMERGENCY_ALARM    BIT(12)
+
+/* Unicast Announcement Type, Generic Audio */
+#define BT_AUDIO_UNICAST_ANNOUNCEMENT_GENERAL    0x00
+#define BT_AUDIO_UNICAST_ANNOUNCEMENT_TARGETED   0x01
```

Comment:
I don't think it's quite right to have a commit with "samples" in the title updating a public header file like this. This change is definitely related, and should therefore be part of the same PR, but it's really an independent dependency to the sample code changes, and should therefore be moved as a separate commit prior to the samples commit.

Reply:
> I don't think it's quite right to have a commit with "samples" in the title updating a public header file like this. This change is definitely related, and should therefore be part of the same PR, but it's really an independent dependency to the sample code changes, and should therefore be moved as a separate commit prior to the samples commit.

Agree, will have a separate commit for this.